doc/proposals/rlp-target-gateway-resource.md

```diff
@@ -52,18 +54,58 @@ spec:
         - conditions: ["admin == yes"]
           max_value: 500
           seconds: 30
-          variables: []       
+          variables: []
 ```
 
 **Note:** No `namespace`/`domain` defined. Kuadrant controller will figure out.
 **Note:** There is no `PREAUTH`, `POSTAUTH` stage defined. Ratelimiting filter should be placed after authorization filter to enable authenticated rate limiting. In the future, `stage` can be implemented.
 
-## Kuadrant-controller's behavior 
-Multiple RLP objects can have references to a given Gateway. Similarly, multiple RLP objects can have references to a given HTTPRoute. The kuadrant controller will iterate over the referenced gateways and aggregate all the rate limit policies that apply for each gateway, including RLP targeting HTTPRoutes. 
+## Kuadrant-controller's behavior
+
+Multiple RLP objects can have references to a single Gateway.
+Similarly, multiple RLP objects can have references to a single HTTPRoute.
```

Comment:
I thought we decided on only one RLP per GWAPI object, right? I remember it was part of the policy attachment doc as well.

Reply:
Yes, but I think we can implement that. Nothing prevents us for doing that. 

Take these words with care because we find some unexpected behavior or over complication and we need to fall back to single RLP per HTTPRoute and Gateway constraint. Evil is in the details.

Reply:
Of course, we can implement but I don't feel comfortable opening that door right now. Opening for RLP suggests considering the same for AP as well then.